exercises/bowling/bowling_test.exs

```diff
@@ -0,0 +1,152 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("bowling.exs")
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule BowlingTest do
+  use ExUnit.Case
+
+  def roll_from_array(game, []) do game end
```

Comment:
Actually in ExUnit I more often see helper functions at the top. See https://github.com/elixir-lang/ecto/blob/master/test/ecto/adapters/postgres_test.exs and other ecto tests for examples.